CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.4.0 (In Progress)
+===================
+
+New Features
+------------
+
+- In-Band-Turn - Previously twilio-video used to send an http request to Endpoint Configuration Service (ECS) to get RTCIceServers before connecting to a Room. The SDK will now get the RTCIceServers directly from the Room Service. This simplifies the connection flow and does not need these [additional connectivity requirements](https://www.twilio.com/docs/video/ip-address-whitelisting#additional-connectivity-requirements) anymore. Also with this change `ConnectOptions` properties `abortOnIceServersTimeout` and `iceServersTimeout` are not needed anymore. (JSDK-2675)
```

Comment:
@makarandp0 I think we should only mention the customer-facing changes here. Please check with @ceaglest .

Reply:
@makarandp0 Also, please merge the changes from master to develop before merging this PR.

Reply:
@manjeshbhargav, I think these are customer kind of customer facing. But I am open to any changes. @ceaglest is OOF today, and I would like get this merged to cut an RC today.

Reply:
@makarandp0 We should start the sentence with the customer facing change. Something like this:
```
The ConnectOptions properties `abortOnIceServersTimeout` and `iceServersTimeout`
are no longer applicable and will be ignored by twilio-video.js. From now on, ICE servers
will be obtained while joining a Room and not by requesting a separate endpoint. (JSDK-2675)
```